R/check.R

```diff
@@ -309,13 +309,13 @@ check_numeric_timestep <- function(dates, date_var, timestep, exact = TRUE) {
     check <- sum(diffs %% timestep) == 0
   }
 
-  if (!check) {
+  if (check) {
+    return(invisible(NULL))
+  }else {
```

Comment:
```suggestion
  } else {
```

Reply:
the linter must complain about that, yeah?

Reply:
Oddly no it doesn't but I have fixed.